src/pages/student_engagement.tsx

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import { useRef } from 'react';
+import Autoplay from 'embla-carousel-autoplay';
+import { Carousel } from '@mantine/carousel';
+
+
+export default function StudentEngagementPage() {
+    const autoplay = useRef(Autoplay({ delay: 2000 }));
+  return (
+    <div style={{ margin: '0 10%'}}>
+      <h1 style={{ textAlign: 'center' }}>STUDENT ENGAGEMENT</h1>
+
+      <div>
+        <h3 style={{ textAlign: 'left', fontSize: '1.2em' }}>WHAT ABOUT US</h3>
+        <p>
+          At Stamford International University, our Student Engagement Team is the heartbeat of the student experience. Committed to creating an environment where every student feels connected, inspired, and supported, our team is here to guide you through a transformative journey.
+        </p>
+        <p>
+          <strong>Student Engagement Advisors:</strong> Your go-to mentors, these advisors are passionate about your success. From academic advice to personal development, they are here to ensure that you make the most out of your time at Stamford.
+        </p>
+        <p>
+          <strong>Event Coordinators:</strong> The architects of unforgettable experiences, our event coordinators work tirelessly to bring you a calendar filled with exciting events, workshops, and activities. Get ready for a dynamic campus life that goes beyond textbooks!
+        </p>
+        <p>
+          <strong>Club Facilitators:</strong> These individuals are the driving force behind our diverse range of clubs and societies. They provide guidance, resources, and a platform for you to explore and express your passions. Whatever your interest, there's a club waiting for you.
+        </p>
+        <p>
+          <strong>Community Outreach Team:</strong> Committed to making a positive impact beyond our campus, this team spearheads community service initiatives, partnerships, and outreach programs. Join us in making a difference in the world.
+        </p>
+      </div>
+
+      <div style={{ textAlign: 'center' }}>
+        <h2 style={{ textAlign: 'center', marginTop: '10%'}} >OUR PAST ACTIVITIES</h2>
+
+
+
+        <div style={{ display: 'flex', alignItems: 'center' }}>
```

Comment:
Use [flex](https://tailwindcss.com/docs/flex) and [align center](https://tailwindcss.com/docs/align-items)